AcadeMeData/migrations/0001_initial.py

```diff
@@ -22,4 +22,14 @@ class Migration(migrations.Migration):
                              ('BA', 'Business Administration'), ('UN', 'Unknown')], default='UN', max_length=2)),
             ],
         ),
+
+        migrations.CreateModel(
+            name='Degree',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('degree', models.CharField(max_length=100)),
```

Comment:
Since you added to the same model file the Degree 
I think you should remove the 0001_initial.py and regenrat it

Reply:
It doesn't work, when I have two different test date in my migration folder (0007_test in additional to the existing 0002_test) I'm getting an error of conflicting migrations..

Reply:
did you try dependencies? 
https://docs.djangoproject.com/en/3.2/howto/writing-migrations/